### ISSUE TITLE
InternalError (see above for traceback): Blas SGEMM launch failed : m=173056, n=32, k=64

### ISSUE THREAD

**Issue by hitzht** [aloyschen/tensorflow-yolo3#21]
hi, running your implementation occurs this error,as follow:
InternalError (see above for traceback): Blas SGEMM launch failed : m=173056, n=32, k=64
	 [[node darknet53/conv2d_3/Conv2D (defined at /ZFS4T/hitzht/tensorflow-yolo3-master-voc/model/yolo3_model.py:109)  = Conv2D[T=DT_FLOAT, _class=["loc:@darknet53/batch_normalization_3/cond/FusedBatchNorm_1/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](darknet53/LeakyRelu_1, darknet53/conv2d_3/kernel/read)]]
	 [[{{node while/strided_slice_1/stack/_615}} = _HostRecv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_3963_while/strided_slice_1/stack", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"](^_cloopwhile/strided_slice_2/stack_2/_525)]]
i try to run the train() with CPU,the error is disappear.hope you give me some advice.thanks

**Comment by LiKangyuLKY** [comment]
Which is your graphics card? some graphics card need to patch for the CUDA.
for example,if your CUDA is 9.0 ,you can browse the page below to put 4 patchs。
`https://developer.nvidia.com/cuda-90-download-archive?target_os=Windows&target_arch=x86_64&target_version=10&target_type=exenetwork`
